### ISSUE TITLE
python cloudflair.py error

### ISSUE THREAD

**Issue by ImLearningHowToCode** [christophetd/CloudFlair#48]
$ python cloudflair.py --help
Traceback (most recent call last):
  File "cloudflair.py", line 3, in <module>
    import dns_utils
  File "/home/kali/CloudFlair/dns_utils.py", line 1, in <module>
    import dns.resolver
ImportError: No module named dns.resolver

I followed every step and everything. It is not working help pls

**Comment by christophetd** [comment]
Can you share the exact commands you ran? Did you install the dependencies via `pip install -r requirements.txt`?

**Comment by Juancollado2003** [comment]
Many issues are resolved just by doing:

sudo apt-get install libxml2-dev libxslt-dev python-dev

**Comment by christophetd** [comment]
I updated the instructions in the README. Can you try the following and confirm it works fine?

```
cd cloudflair
python3 -m venv venv
source venv/bin/activate
pip install -r requirements.txt
python cloudflair.py google.com
```

**Comment by mrsnax** [comment]
Its work but  got warning

```
cloudflair.py:35: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if len(cert_fingerprints) is 0:
cloudflair.py:44: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if len(hosts) is 0:
cloudflair.py:140: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if len(origins) is 0:
[*] Retrieving Cloudflare IP ranges from https://www.cloudflare.com/ips-v4
[*] The target appears to be behind CloudFlare.
[*] Looking for certificates matching "garudaku.com" using Censys
[*] 0 certificates matching "garudaku.com" found.
```

> I updated the instructions in the README. Can you try the following and confirm it works fine?
> 
> ```
> cd cloudflair
> python3 -m venv venv
> source venv/bin/activate
> pip install -r requirements.txt
> python cloudflair.py google.com
> ```

**Comment by christophetd** [comment]
Fixed by #49
